boards/hifive1/src/main.rs

```diff
@@ -36,7 +32,13 @@ const FAULT_RESPONSE: kernel::procs::FaultResponse = kernel::procs::FaultRespons
 
 // RAM to be shared by all application processes.
 #[link_section = ".app_memory"]
-static mut APP_MEMORY: [u8; 5 * 1024] = [0; 5 * 1024];
+static mut APP_MEMORY: [u8; 6 * 1024] = [0; 6 * 1024];
+
+// Force the emission of the `.apps` segment in the kernel elf image
+// NOTE: This will cause the kernel to overwrite any existing apps when flashed!
+#[used]
+#[link_section = ".app.hack"]
+static APP_HACK: u8 = 0;
```

Comment:
Do we need this? It is quite annoying to have to re-flash apps all the time.

Reply:
Removed